tests/io/test_shell.py

```diff
@@ -211,17 +211,6 @@ def test_check_convert_qresources(self, shell_io):
         qr = QResources()
         assert shell_io.check_convert_qresources(qr) == {}
 
-    def test_convert_qresources(self, shell_io):
-        qr = QResources(processes=1)
-        with pytest.raises(
-            UnsupportedResourcesError,
-            match=r"Only empty QResources is supported",
-        ):
-            shell_io._convert_qresources(qr)
-
-        qr = QResources()
-        assert shell_io._convert_qresources(qr) == {}
-
```

Comment:
I'd probably adapt the test here no ?

Reply:
I added a specific test in the `test_check_convert_qresources`. The test for `_convert_qresources` was basically just testing the raise, which has been removed.